platform/Dockerfiles/virtiofsd/Dockerfile

```diff
@@ -148,65 +104,38 @@ RUN make install
 #
 # Prepare final application
 
-FROM dependencies as build_virtiofsd
+FROM dependencies AS build_virtiofsd
 ENV PATH                      "$PATH:/root/.cargo/bin/"
-ENV RUST_VERSION              1.74.0
+ENV RUST_VERSION              1.80.0
 ENV RUST_ARCH                 x86_64
-ENV VIRTIOFSD_VERSION         v1.10.1
+ENV VIRTIOFSD_VERSION         1.11.1
 ENV RUSTFLAGS                 "--codegen target-feature=+crt-static --codegen link-self-contained=yes"
 ENV LIBSECCOMP_LINK_TYPE      "static"
 ENV LIBSECCOMP_LIB_PATH       "/usr/lib"
 ENV LIBCAPNG_LINK_TYPE        "static"
 ENV LIBCAPNG_LIB_PATH         "/usr/lib"
-# -- -D warnings -D clippy::undocumented_unsafe_blocks
-# ENV RUSTFLAGS               "-D warnings -D clippy::undocumented_unsafe_blocks"
-# ENV RUSTFLAGS               "-C link-arg=-fuse-ld=lld"
-# or with Cargo's configuration file (i.e see .cargo/config.toml).
+
+
+###
+#
+# Build virtiofsd binary and debian package
 
 WORKDIR /workspace/build/
-RUN curl -l https://gitlab.com/virtio-fs/virtiofsd/-/archive/${VIRTIOFSD_VERSION}/virtiofsd-${VIRTIOFSD_VERSION}.tar.gz --output virtiofsd-${VIRTIOFSD_VERSION}.tar.gz
-RUN tar -xf virtiofsd-${VIRTIOFSD_VERSION}.tar.gz
+RUN git clone --depth 1 --branch v${VIRTIOFSD_VERSION} https://gitlab.com/virtio-fs/virtiofsd virtiofsd-v${VIRTIOFSD_VERSION}
+
+WORKDIR /workspace/build/"virtiofsd-v${VIRTIOFSD_VERSION}"
+# Need the virtiofsd features. --all-features was here prior
+RUN cargo deb --profile release --target=${RUST_ARCH}-unknown-linux-musl --separate-debug-symbols --compress-debug-symbols -- --all-features
```

Comment:
this may not actually compile virtiofsd with `--all-features`. The build prints an error but completes and produces artifacts. Will need to verify the build after commit as we have no pre-commit checks.